Controllers/DayOffRequestController.cs

```diff
@@ -12,40 +13,33 @@ namespace stunning_robot_HR.Controllers
 {
     public class DayOffRequestController : Controller
     {
-        private readonly stunning_robot_HRContext _context;
-
+        private stunning_robot_HRContext _context;
```

Comment:
Man does this controller look good. Clean, loosely coupled. If we wanted to start saving and retrieving data from MongoDB tomorrow, this class wouldn't change at all.